### PR TITLE
Update ovms_quickstart.md

### PR DESCRIPTION
**Pull request by jsxyhelu**
The original command line method does not work on my computer. First put the models according to the specified structure, and then use the command line I modified to use.

### CODE REVIEW

File: docs/ovms_quickstart.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.4/op
 Start the Model Server container:
 
 ```bash
-docker run -d -v <folder_with_downloaded_model>:/models/face-detection/1 -p 9000:9000 openvino/model_server:latest \
+docker run -d -v <folder_with_downloaded_model_in_a_defined_structur>:/models:ro  -p 9000:9000 openvino/model_server:latest \
```
Comment (pengguanjun) [inline docs/ovms_quickstart.md:64]: I test the method,but I cann't work it.

[](https://www.cnblogs.com/jsxyhelu/p/13796161.html?tdsourcetag=s_pctim_aiomsg#commentform)

My test step4:
[root@VM-0-15-centos ~]# `docker run -v /models:/models:ro -p 9000:9000 openvino/model_server:latest --model_path /models/model1 --model_name face-detection --port 9000 --log_level DEBUG --shape auto`
[2020-10-12 14:57:36.726] [serving] [debug] CLI parameters passed to ovms server
[2020-10-12 14:57:36.726] [serving] [debug] model_path: /models/model1
[2020-10-12 14:57:36.726] [serving] [debug] model_name: face-detection
[2020-10-12 14:57:36.726] [serving] [debug] batch_size: 0
[2020-10-12 14:57:36.726] [serving] [debug] shape: auto
[2020-10-12 14:57:36.726] [serving] [debug] model_version_policy: 
[2020-10-12 14:57:36.726] [serving] [debug] nireq: 0
[2020-10-12 14:57:36.726] [serving] [debug] target_device: CPU
[2020-10-12 14:57:36.726] [serving] [debug] plugin_config: 
[2020-10-12 14:57:36.726] [serving] [debug] gRPC port: 9000
[2020-10-12 14:57:36.726] [serving] [debug] REST port: 0
[2020-10-12 14:57:36.726] [serving] [debug] REST workers: 24
[2020-10-12 14:57:36.726] [serving] [debug] gRPC workers: 1
[2020-10-12 14:57:36.726] [serving] [debug] gRPC channel arguments: 
[2020-10-12 14:57:36.726] [serving] [debug] log level: DEBUG
[2020-10-12 14:57:36.726] [serving] [debug] log path: 
[2020-10-12 14:57:36.726] [serving] [debug] Batch size set: false, shape set: true
[2020-10-12 14:57:36.726] [serving] [debug] Currently registered versions count:0
[2020-10-12 14:57:36.726] [serving] [info] Getting model from /models/model1
[2020-10-12 14:57:36.726] [serving] [info] Model downloaded to /models/model1
[2020-10-12 14:57:36.726] [serving] [info] Will add model: face-detection; version: 1 ...
[2020-10-12 14:57:36.726] [serving] [info] Loading model: face-detection, version: 1, from path: /models/model1/1, with target device: CPU ...
[2020-10-12 14:57:36.726] [serving] [info] Some inputs shapes for model face-detection are set to auto
[2020-10-12 14:57:36.726] [serving] [info] STATUS CHANGE: Version 1 of model face-detection status change. New status: ( "state": "START", "error_code": "OK" )
[2020-10-12 14:57:36.726] [serving] [debug] setLoading: face-detection - 1 (previous state: START) -> error: OK
[2020-10-12 14:57:36.726] [serving] [info] STATUS CHANGE: Version 1 of model face-detection status change. New status: ( "state": "LOADING", "error_code": "OK" )
[2020-10-12 14:57:36.726] [serving] [debug] Getting model files from path:/models/model1/1
[2020-10-12 14:57:36.726] [serving] [error] Could not find *.bin file for model:face-detection version:1 in path:/models/model1/1
[2020-10-12 14:57:36.726] [serving] [debug] setLoading: face-detection - 1 (previous state: LOADING) -> error: UNKNOWN
[2020-10-12 14:57:36.726] [serving] [info] STATUS CHANGE: Version 1 of model face-detection status change. New **status: ( "state": "LOADING", "error_code": "UNKNOWN" )**
[2020-10-12 14:57:36.727] [serving] [error] Error occurred while loading model: face-detection; version: 1; error: File not found or cannot open
[2020-10-12 14:57:36.727] [serving] [error] Error occurred while loading model: face-detection versions; error: File not found or cannot open
[2020-10-12 14:57:36.727] [serving] [error] Couldn't start model manager
[2020-10-12 14:57:36.727] [serving] [error] ovms::ModelManager::Start() Error: File not found or cannot open
[root@VM-0-15-centos ~]# `tree`
.
|-- client_requirements.txt
|-- client_utils.py
|-- face_detection.py
|-- images
|   `-- people1.jpeg
|-- model
|   |-- face-detection-retail-0004.bin
|   |-- face-detection-retail-0004.xml
|   `-- models
|       `-- model1
|           |-- 1
|           |   |-- face-detection-retail-0004.bin
|           |   `-- face-detection-retail-0004.xml
|           |-- face-detection-retail-0004.bin
|           `-- face-detection-retail-0004.xml
|-- --model_path
|-- __pycache__
|   `-- client_utils.cpython-36.pyc
`-- results

7 directories, 12 files

Comment (pengguanjun) [reply docs/ovms_quickstart.md:64]: 图片

Comment (dtrawins) [reply docs/ovms_quickstart.md:64]: @pengguanjun it looks to me like the problem is related to permissions on the model files. OVMS container is started by default with the security context of ovms account with uid 5000. If your model files are restricted with permissions 750, the ovms account will not have read access to load the models.
There are two solutions available - one is to change the model folder and files permissions to 755 or change the security context of the OVMS docker container to root or your personal account like `--user root` or `--user $(id -u):$(id -g) `
Probably there was the same root cause with the original problem reported by @jsxyhelu

Comment (pengguanjun) [reply docs/ovms_quickstart.md:64]: @dtrawins 
Thanks.
I have a try,I don't get the good way to use the --user root instruction.


[root@VM-0-15-centos ~]# `docker run -v /root/models/face-detection/2:/models/face-detection/1/1 -p 9000:9000 openvino/model_server:latest --model_path /models/face-detection/1 --model_name face-detection --port 9000 --log_level DEBUG --shape auto **--user roo**t`
error parsing options: Option ‘user’ does not exist

Hope to get specific guidance, thanks.

Comment (dtrawins) [reply docs/ovms_quickstart.md:64]: the `--user root` would be the docker parameter:
`docker run --user root -v /root/models/face-detection/2:/models/face-detection/1/1 -p 9000:9000 openvino/model_server:latest --model_path /models/face-detection/1 --model_name face-detection --port 9000 --log_level DEBUG --shape auto`

**Comment by dtrawins** [comment]
@pengguanjun the logs you included indicate that model is loaded just fine from folder /root/models/face-detection/2.
Log line `Currently registered versions count:` every 1s is from a regular check for new model version. You could disable it with a parameter --file_system_poll_wait_seconds=0
With your models folder structure, you could also simplify the docker command like:
`docker run -v /root/models/face-detection:/models/face-detection -p 9000:9000 openvino/model_server:latest --model_path /models/face-detection/ --model_name face-detection --port 9000 --log_level DEBUG --shape auto`

**Comment by pengguanjun** [comment]
@dtrawins 

Thanks，you are handsome man.
It is good.

图片

**Comment by dtrawins** [comment]
Quick start guide has been updated as part of PR https://github.com/openvinotoolkit/model_server/pull/314 .
@jsxyhelu Thank you for your contribution.

**Comment by leo-XUKANG** [comment]
@dtrawins  thanks you for your contribution

**Pull request closed without merging**